redaxo/src/addons/media_manager/lib/effects/effect_jpg_quality.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @package redaxo\media-manager
+ */
+class rex_effect_jpg_quality extends rex_effect_abstract
+{
+    public function execute()
+    {
+        $this->image->img['quality'] = $this->params['quality'];
```

Comment:
Hast du den Effekt getestet? `$this->image` scheint es nicht mehr zu geben, sondern `$this->media`.
Wobei das wiederum keine `img`-Property hat.
Weiß gerade gar nicht so genau, wie man nun die Quality setzen könnte.